content/billing/pricing.md

```diff
@@ -41,6 +41,8 @@ Usage on macOS standard VM that exceeds 500 minutes is charged at rate shown bel
 
 Builds on macOS premium VM, Linux standard VM, and Linux premium VM do not have free build minutes and are charged at the rates shown below. 
 
+Each extra build concurrency allows to run an additional build in paralel. For example, two extra build concurrencies allow to run three builds in paralel. Consider Business plan or Enterprise plan if more than three concurrent builds are required.
```

Comment:
The build concurrency information should not be under Pricing for personal accounts but under Pricing for teams. 
Also need to update line 10 and say that additional concurrency is available for teams on the pay as you go plan.